terratest/src/test/pega/utilities.go

```diff
@@ -64,10 +64,6 @@ func VerifyInitContainerData(t *testing.T, containers []k8score.Container, optio
 			require.Equal(t, "cassandra:3.11.3", container.Image)
 			//The cassandra svc name below is derived from helm release name and not .Values.global.deploymentName like search svc
 			require.Equal(t, []string{"sh", "-c", "until cqlsh -u \"dnode_ext\" -p \"dnode_ext\" -e \"describe cluster\" pega-cassandra 9042 ; do echo Waiting for cassandra to become live...; sleep 10; done;"}, container.Command)
-		} else if name == "wait-for-cassandra" {
```

Comment:
I just removed this block since I noticed it was a duplicate of the one above it